easybuild/framework/easyblock.py

```diff
@@ -1239,15 +1272,27 @@ def fetch_step(self, skip_checksums=False):
                 raise EasyBuildError("EasyBuild-version %s is newer than the currently running one. Aborting!",
                                      easybuild_version)
 
+        if build_option('extended_dry_run'):
+
+            print_msg("Available download URLs for missing sources/patches:", silent=self.silent, prefix=False)
```

Comment:
I would change this to:

"Available download URLs for required sources/patches:"

I think I know why you've phrased it as "missing", but it sounds odd.